### PR TITLE
Checkbox+Button

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

File: docs/library/forms/button.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    "Like",
+    color_scheme="red",
+)
+```
+
+## Props
+
+### Disabled
+
+The `disabled` prop disables the button, by default it is `False`.
```
Comment (masenf) [inline docs/library/forms/button.md:32]:
```suggestion
The `disabled` prop disables the button, by default it is `False`. A disabled button does not respond to user interactions such as click and cannot be focused.
```

File: docs/library/forms/button.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    count: int = 0
+
+    def increment(self):
+        self.count += 1
+
+    def decrement(self):
+        self.count -= 1
+
+def counter():
+    return rx.hstack(
```
Comment (masenf) [inline docs/library/forms/button.md:102]: we should use the radix flex with `justify="between"`

File: docs/library/forms/button.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        rdxt.button(
+            "Decrement",
+            color_scheme="red",
+            on_click=CountState.decrement,
+        ),
+        rx.heading(
```
Comment (masenf) [inline docs/library/forms/button.md:108]:
```suggestion
        rdxt.heading(
```

File: docs/library/forms/checkbox.md [hunk 4]
```diff
@@ -5,26 +5,124 @@ components:
...
+    rdxt.checkbox(),
+    rdxt.checkbox(disabled=True),
+)
+```
+
+## Triggers
+
+### OnCheckedChange
+
+The `on_change` trigger is called when the checkbox is clicked.
```
Comment (masenf) [inline docs/library/forms/checkbox.md:48]:
```suggestion
The `on_checked_change` trigger is called when the checkbox is clicked.
```

File: docs/library/forms/checkbox.md [hunk 5]
```diff
@@ -5,26 +5,124 @@ components:
...
+
+```python demo
+rdxt.checkbox(default_checked=True, on_checked_change=rx.window_alert("Checked!"))
+```
+
+## Styling
+
+### Color
+
+The `color` prop changes the color of the checkbox. The default color is `primary`.
```
Comment (masenf) [inline docs/library/forms/checkbox.md:58]:
```suggestion
### Color Scheme

The `color_scheme` prop changes the color of the checkbox. The default color maps to the `accent_color` in the app's `theme`.
```

File: docs/library/forms/checkbox.md [hunk 6]
```diff
@@ -5,26 +5,124 @@ components:
...
+      rdxt.checkbox(color_scheme="orange", default_checked=True, high_contrast=True),
+      rdxt.checkbox(color_scheme="tomato", default_checked=True, high_contrast=True),
+  )
+)
+```
+
+
+### Size
+
+The `size` prop changes the size of the button. The default size is `2`.
```
Comment (masenf) [inline docs/library/forms/checkbox.md:91]:
```suggestion
The `size` prop changes the size of the checkbox. The default size is `2`.
```

**Review by masenf — CHANGES_REQUESTED** [review]
we should avoid the chakra hstack and vstack components in the new docs.

File: pcweb/pcweb.py [hunk 7]
```diff
@@ -35,6 +36,20 @@
         image="/previews/index_preview.png",
     )
 
+for key in chakra_components:
```
Comment (picklelo) [inline pcweb/pcweb.py:39]: Shouldn't need the changes in this file

**Pull request closed without merging**